apps/backoffice-v2/src/common/utils/svg-to-png/svg-to-png.ts

```diff
@@ -0,0 +1,23 @@
+export const svgToPng = (imageUrl: string): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    const img = new Image();
+    img.crossOrigin = 'Anonymous';
+    img.onload = () => {
+      const canvas = document.createElement('canvas');
+      const context = canvas.getContext('2d');
+
+      canvas.width = img.width;
+      canvas.height = img.height;
+      context?.drawImage(img, 0, 0);
+
+      const dataUrl = canvas.toDataURL('image/png');
+
+      resolve(dataUrl);
+      document.body.removeChild(canvas);
```

Comment:
Don't see we are adding the canvas to the body to begin with.

Reply:
Dumb copilot. Removed.